webapp/src/projects.tsx

```diff
@@ -747,6 +747,12 @@ export class ExitAndSaveDialog extends data.Component<ISettingsProps, ExitAndSav
 
     handleChange(name: string) {
         this.setState({ projectName: name });
+        let header = document.getElementsByClassName("exitandsave").item(0).firstChild as HTMLElement;
+        if (name === "" || name === "Untitled") {
```

Comment:
Also, Untitled is not the name of the project in all languages. 
Add lf around that to get the Untitled name in the user's lang